src/night_light/silver_db/median_income.py

```diff
@@ -0,0 +1,25 @@
+from night_light.bronze_db.util import connect_to_duckdb
+
+def generate_silver_median_income() -> None: 
+    """
+    Creates median income table for the silver database
+
+    Rewrites pervious verison of the table if it exists.
+    """
+    db_path = "silver.db"
+    conn = connect_to_duckdb(db_path)
+    conn.execute("ATTACH DATABASE 'bronze.db' AS bronze")
+
+    query = """
+    DROP TABLE IF EXISTS crosswalks;
```

Comment:
Why does it drop the `crosswalks` table? Was it supposed to drop `median_income` table if it already exists?